homeassistant/components/wmspro/cover.py

```diff
@@ -32,25 +32,29 @@ async def async_setup_entry(
     entities: list[WebControlProGenericEntity] = []
     for dest in hub.dests.values():
         if dest.action(WMS_WebControl_pro_API_actionDescription.AwningDrive):
-            entities.append(WebControlProAwning(config_entry.entry_id, dest))  # noqa: PERF401
+            entities.append(WebControlProAwning(config_entry.entry_id, dest))
+        elif dest.action(
+            WMS_WebControl_pro_API_actionDescription.RollerShutterBlindDrive
+        ):
+            entities.append(WebControlProRollerShutter(config_entry.entry_id, dest))
```

Comment:
Since you have tests, it would be nice if we can also cover this with tests

Reply:
Yes, certainly. Will try to implement them.

Reply:
You can use parametrize. I believe pytest has a way to get fixtures dynamically with a fixture requester, I don't have the exact name now